Intersect.Client/MonoGame/Graphics/MonoRenderer.cs

```diff
@@ -682,12 +674,20 @@ private static double GetDistance(double x1, double y1, double x2, double y2)
             return root;
         }
 
-        private Pointf Rotate(Pointf pnt, Pointf ctr, float angle)
+        private void Rotate(ref float x, ref float y, float centerX, float centerY, float angle)
         {
-            return new Pointf(
-                (float) (pnt.X + ctr.X * Math.Cos(angle) - ctr.Y * Math.Sin(angle)),
-                (float) (pnt.Y + ctr.X * Math.Sin(angle) + ctr.Y * Math.Cos(angle))
-            );
+            // Rotate the point around the center point.
+            float s = (float)Math.Sin(angle);
+            float c = (float)Math.Cos(angle);
+
+            x -= centerX;
+            y -= centerY;
+
+            float newX = x * c - y * s;
+            float newY = x * s + y * c;
+
+            x = newX + centerX;
+            y = newY + centerY;
```

Comment:
Change it back

Reply:
see https://github.com/AscensionGameDev/Intersect-Engine/pull/1702/files#r1060247028

Reply:
[same](https://github.com/AscensionGameDev/Intersect-Engine/pull/1702#discussion_r1060807951) for this